app/src/main/java/ee/mcdimus/androidacademyapp/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,37 @@
+package ee.mcdimus.androidacademyapp
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesDetails : Fragment() {
+
+  private var navigationListener: FragmentNavigationListener? = null
+
+  override fun onAttach(context: Context) {
+    super.onAttach(context)
+
+    if (context is FragmentNavigationListener) {
+      navigationListener = context
+    }
```

Comment:
Можно написать чуть-чуть лаконичнее
```suggestion
    navigationListener = context as? FragmentNavigationListener
```